sdk/resources/Azure.ResourceManager.Resources/src/Generated/Models/AzureCliScript.Serialization.cs

```diff
@@ -117,7 +117,7 @@ internal static AzureCliScript DeserializeAzureCliScript(JsonElement element)
             ResourceIdentifier id = default;
             string name = default;
             ResourceType type = default;
-            SystemData systemData = default;
+            Optional<SystemData> systemData = default;
```

Comment:
why does this result in only serialization changes without changing any base types in resources package?

Reply:
Previously, during replacement with the common type, we hardcoded `Required` as true for its properties. I have updated generator to fix that. See SystemObjectType code [here](https://github.com/Azure/autorest.csharp/pull/2303/files#diff-32a4ccc7bb0ed76e7043f8ac39b8b80af8dcbf850aeb1add76ccd922b6215c8e).